### PR TITLE
Migrate from Ubuntu 16.04 to 20.04

### PR DESCRIPTION
**Pull request by kit1980**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -13,9 +13,6 @@ jobs:
       - run:
           name: Install Conda
           command: ./scripts/install_basics.sh
-      - run:
-          name: Install CUDA 11.3
-          command: sudo ./scripts/install_cuda.sh
```
Comment (xuzhao9) [inline .circleci/config.yml:18]: Can you also help remove the `scripts/install_cuda.sh`, since it is not used?

**Review by xuzhao9 — APPROVED** [review]
LGTM - please also remove `install_cuda.sh`. Also, what cuda and cudnn version does this image use?

File: .circleci/config.yml (hunk 1 above)
Comment (kit1980) [reply .circleci/config.yml:18]: removed

Comment (kit1980) [reply .circleci/config.yml:18]: @xuzhao9 CUDA version on the host is 11.4
CuDNN is not installed on the host, but it's bundled with PyTorch, right?
`torch.backends.cudnn.version()` returns  8302, so CuDNN version 8.3.2.

Comment (xuzhao9) [reply .circleci/config.yml:18]: @kit1980 Thanks for the information. Yes CUDA 11.4 and CuDNN 8.3.2 looks good (should be compatible with PyTorch cu113 build).
Right, CuDNN is now bundled with PyTorch (https://github.com/pytorch/builder/pull/1005).

**Pull request closed without merging**